fdbserver/GrvProxyServer.actor.cpp

```diff
@@ -302,6 +304,133 @@ ACTOR Future<Void> healthMetricsRequestServer(GrvProxyInterface grvProxy,
 	}
 }
 
+ACTOR Future<Void> globalConfigMigrate(GrvProxyData* grvProxyData, GlobalConfigMigrateRequest req) {
+	state Key migratedKey("\xff\x02/fdbClientInfo/migrated/"_sr);
+	state Reference<ReadYourWritesTransaction> tr;
+	try {
+		state Backoff backoff;
```

Comment:
I think we can go back to using transactions in the standard way now. The problem before was that we had a reference cycle: the database had a global config that had a transaction that referred to the database. Now, this is just an ordinary transaction scoped to the request.